benchmark/benchmark.cpp

```diff
@@ -109,71 +100,67 @@ UMF_BENCHMARK_TEMPLATE_DEFINE(alloc_benchmark, scalable_pool_fix,
                               pool_allocator<scalable_pool<os_provider>>);
 
 UMF_BENCHMARK_REGISTER_F(alloc_benchmark, scalable_pool_fix)
-    ->Args({10000, 0, 4096})
-    ->Args({10000, 100000, 4096})
-    ->Threads(4)
-    ->Threads(1);
+    ->Apply(&default_alloc_fix_size);
 
 UMF_BENCHMARK_TEMPLATE_DEFINE(alloc_benchmark, scalable_pool_uniform,
                               uniform_alloc_size,
                               pool_allocator<scalable_pool<os_provider>>);
 
 UMF_BENCHMARK_REGISTER_F(alloc_benchmark, scalable_pool_uniform)
-    ->Args({10000, 0, 8, 64 * 1024, 8})
-    ->Threads(4)
-    ->Threads(1);
+    ->Apply(&default_alloc_uniform_size);
 #endif
 // Multiple allocs/free
+static void
+default_multiple_alloc_fix_size(benchmark::internal::Benchmark *benchmark) {
+    benchmark->Args({10000, 4096});
+    benchmark->Threads(4);
+    benchmark->Threads(1);
+}
+
+static void
+default_multiple_alloc_uniform_size(benchmark::internal::Benchmark *benchmark) {
+    benchmark->Args({10000, 0, 8, 64 * 1024, 8});
```

Comment:
fyi, some of these before was `({10000, 8, 64 * 1024, 8})` (one extra `0` here) - is that expected? ;)

Reply:
nope - this was copy paste error - nice catch